scripts/hid_configurator/led_stream.py

```diff
@@ -65,5 +75,187 @@ def send_continuous_led_stream(dev, recipient, DEVICE_CONFIG, led_id, freq, subs
     print('LED stream ended')
 
 
+def peak_to_hue(peak):
+    assert peak >= 0 and peak <= 1
+
+    G = 120
+    B = 240
+    HUE_MAX = 360
+
+    hue = (B - peak * (B - G))
+    hue /= HUE_MAX
+
+    return hue
+
+
+def gen_led_color(data):
+    # Cast to prevent overflow
+    peak = np.abs(np.int32(np.max(data)) - np.min(data)) / np.iinfo(np.int16).max
+    if peak > 1:
+        peak = 1
+    hue = peak_to_hue(peak)
+
+    return colorsys.hsv_to_rgb(hue, 1, 1)
+
+
+def send_led_effects(send_error_event, queue):
+    while True:
+        data = queue.get()
+
+        if data == None:
+            # LED stream ended
+            return
+
+        data = np.frombuffer(data, dtype=np.int16)
+
+        duration = send_led_effects.BASE_DURATION + send_led_effects.reminder +\
+                   send_led_effects.duration_increment
+        r, g, b = gen_led_color(data)
+        SUBSTEP_CNT = 10
```

Comment:
why name is capital?

Reply:
Capital names are commonly used for constants in Python. This one is constant here.